librosa/core/spectrum.py

```diff
@@ -849,9 +850,15 @@ def power_to_db(S, ref=1.0, amin=1e-10, top_db=80.0):
 
     """
 
+    S = np.array(S)
```

Comment:
Maybe use `np.asarray` to avoid unnecessary copies ? (same below)
`np.array` copies by default, see for example: https://stackoverflow.com/questions/14415741/numpy-array-vs-asarray

Reply:
Thanks for the tip!